app/javascript/packs/admin_timelogs/new_timelog.jsx

```diff
@@ -29,6 +32,7 @@ class NewAdminTimelog extends React.Component {
     Fetch.json(`/api/v1/admin/user/users/${this.state.userId}/timelogs/trello_cards`)
       .then((data) => {
         this.setState({ trelloCards: data });
+        $("#cards_dropdown").prepend("<option value='' selected='selected'></option>");
```

Comment:
I think it is easier to render HTML using HTML. We don't need to do this with js.